traffic_ops/testing/api/v3/jobs_test.go

```diff
@@ -82,6 +85,47 @@ func CreateTestInvalidationJobs(t *testing.T) {
 	}
 }
 
+func CreateTestInvalidJob(t *testing.T) {
+	toDSes, _, err := TOSession.GetDeliveryServicesNullable()
+	if err != nil {
+		t.Fatalf("cannot GET Delivery Services: %v - %v", err, toDSes)
+	}
+	dsNameIDs := map[string]int64{}
+	for _, ds := range toDSes {
+		dsNameIDs[*ds.XMLID] = int64(*ds.ID)
+	}
+
+	job := testData.InvalidationJobs[0]
+	_, ok := dsNameIDs[(*job.DeliveryService).(string)]
+	if !ok {
+		t.Fatalf("can't create test data job: delivery service '%v' not found in Traffic Ops", job.DeliveryService)
+	}
+	maxRevalDays := 0
+	foundMaxRevalDays := false
+	for _, p := range testData.Parameters {
+		if p.Name == "maxRevalDurationDays" {
+			maxRevalDays, err = strconv.Atoi(p.Value)
+			if err != nil {
+				t.Fatalf("unable to parse maxRevalDurationDays value '%s' to int", p.Value)
+			}
+			foundMaxRevalDays = true
+			break
```

Comment:
This block can be unindented if it's moved to after the `if` statement, the condition is negated, and the loop `continue`s if the condition is met

Reply:
unindented in 7509a9faeb20027557d9cc2332755dd47c8e8498